pxe/user-data.template

```diff
@@ -0,0 +1,87 @@
+#cloud-config
+autoinstall:
+  version: 1
+  proxy: http://_gateway:${ACNG_PORT}
+  kernel:
+    flavor: hwe
+  shutdown: reboot
+  identity:
+    hostname: ${MACHINE_HOSTNAME}
+    password: ${MACHINE_PASSWORD}
+    realname: ${MACHINE_USERNAME}
+    username: ${MACHINE_USERNAME}
+  storage:
+    grub:
+      reorder_uefi: true
+      update_nvram: true
+      reorder_uefi_force_fallback: true
+    config:
```

Comment:
Why the disk config? We've discussed this during tuesdays to leave it in right? The extra room left by the default config is awesome for creating snapshots and thus to quickly reset the machines to a clean state.

Reply:
The HP laptops need a FAT boot partition, which the default LVM layout does not provide nor an option to override the filesystem type. However, we switched to UEFI booting after, and the EFI LVM layout does ofcourse use a FAT partition.

I would either only support UEFI booting, or keep the manual layout, to avoid bad installs on the laptops.